### ISSUE TITLE
VoiceOver tells user they tapped on the search box when they tap on the search button

### ISSUE THREAD

**Issue by MikePulsiferDOL** [USDepartmentofLabor/Comply-Chain#230]
VoiceOver tells user they tapped on the search box when they tap on the search button.

https://user-images.githubusercontent.com/1970829/169328798-ce38df6e-8a9d-4890-8e77-1dddbe35325f.mov

**Comment by vikramkm114a** [comment]
Hi Mike, we tested this issue on multiple build versions, devices, as well as the current application in the app store and were unable to replicate your results.

**Comment by MikePulsiferDOL** [comment]
This is clearly not working.  You can see from this video that I am told "Search box" when I am tapping the button.

https://user-images.githubusercontent.com/1970829/170352604-388d7865-5c88-43cd-9a36-cc7f8eea0048.mov

**Comment by jfc3-dol** [comment]
I have had similar issues with VoiceOver as @MikePulsiferDOL had. But I assumed it was more about the devices I was using being older and were causing the problem.

**Comment by jfc3-dol** [comment]
Maybe the problem is where the page focus goes after entering the information in the Search field when using Voiceover. Or that the tap/click area is not located in the correct location. So when a user taps the screen with VoiceOver on for the search button, the application thinks it's the location for the search box. This isn't a good thing, and maybe as a test, if you tap below the Search button, VoiceOver will then read it's the Search button. It's not a good thing, but a way to see what the issue might be.

I will see if I can download the latest version from TestFlight and see what happens on my older devices.

**Comment by jfc3-dol** [comment]
I tested the latest build 1.4 (1.0.55), on both of our test devices, which I downloaded from TestFlight.

When I read the "Search" button with VoiceOver, it reads "Search box button". The button part read by VoiceOver is the function of the item. So it looks like for me, at least the text description is "Search Box" for some reason instead of only being "Search", or it's getting added in somehow.

**Comment by MikePulsiferDOL** [comment]
My original video was an iPhone 13 running iOS 15.5
This video is an iPad Air running iOS 15.5

https://user-images.githubusercontent.com/1970829/170540398-8a5118eb-aa93-4a38-a19f-63401501812a.mov

Multiple taps on the search button where it says "Search box"
